src/css/_popover.scss

```diff
@@ -24,7 +23,7 @@
                     background-color: transparent;
                 }
                 &:not(:last-child) {
-                    border-bottom: 1px solid $color-gray;
+                    border-bottom: 1px solid #808080;
```

Comment:
I know this is right in the middle... but if it were changed, it would need to be changed in two ways... so I'd use a mag-get here, just as a reminder, even if it is degenerate.